api/src/main/java/org/openmrs/Order.java

```diff
@@ -89,6 +89,13 @@
 	private Date scheduledDate;
 	
 	/**
+	 * Allows the orders if ordered as an orderGroup,
+	 * to maintain a sequence of how members are added in the group
+	 * ex - for two orders of isoniazid and ampicillin, the sequence of 1 and 2 needed to be maintained
+	 */
+	private Integer sortWeight;
```

Comment:
Sort weight is a double in [most other cases](http://resources.openmrs.org/datamodel/openmrs-data-model-1.11.html#.sort_weight), so this should probably be `Double`.